src/cfsssh/cloudinit/bss.py

```diff
@@ -52,17 +56,17 @@ class BSSException(Exception):
     This allows us a clean way to retry these interactions.
     """
 
-def get_global_metadata_key(key, session=None):
+def get_global_metadata_key(key: str, session: Optional[requests.Session]=None) -> str:
     session = session or requests_retry_session()
-    get_payload = {'key': 'Global.%s' %(key)}
-    response = session.get(METADATA_ENDPOINT, json=get_payload)
+    get_params = {'key': 'Global'}
```

Comment:
Don't you need to add the 'key' into the value part of the pair here?

Reply:
No -- that is the way the backports are different. BSS as-is cannot handle the parameters if the key value is specified that way. This is being fixed, and the fix will be in CSM 1.7.0. The BSS fix is also being backported, but I didn't want cfs-trust to have a dependency on that, since this method also works fine.